src/resolve/resolved-dnstls-openssl.c

```diff
@@ -51,6 +51,8 @@ static int dnstls_flush_write_buffer(DnsStream *stream) {
 int dnstls_stream_connect_tls(DnsStream *stream, DnsServer *server) {
         _cleanup_(BIO_freep) BIO *rb = NULL, *wb = NULL;
         _cleanup_(SSL_freep) SSL *s = NULL;
+        X509_VERIFY_PARAM *v;
+        const unsigned char *ip;
```

Comment:
doesn't matter much, but please move into the if block where this is needed. we try to keep most variables declared at minimal scope